src/vos/tests/evt_ctl.c

```diff
@@ -127,19 +144,21 @@ ts_open_create(bool create, char *args)
 
 	if (rc != 0) {
 		D_PRINT("Tree %s failed: %d\n", create ? "create" : "open", rc);
-		return -1;
+		fail();
 	}
-	return 0;
 }
 
-static int
-ts_close_destroy(bool destroy)
+static void 
+ts_close_destroy(void **state)
 {
+        bool destroy;
 	int rc;
 
+        destroy = tst_fn_val.input;
+
 	if (daos_handle_is_inval(ts_toh)) {
 		D_PRINT("Invalid tree open handle\n");
-		return -1;
+	        fail();	
```

Comment:
(style)  trailing whitespace
(style)  code indent should use tabs where possible